src/util.ts

```diff
@@ -4,11 +4,19 @@ const execAsPromised = promisify(exec);
 
 import { error, info, setFailed } from "@actions/core";
 
-const execBashCommand = async (command: string): Promise<string> => {
+const execBashCommand = async (
+  command: string,
+  platform: NodeJS.Platform = process.platform
+): Promise<string> => {
+  const shell =
+    platform === "win32"
+      ? "C:\\Program Files\\Git\\bin\\bash.exe"
+      : "/usr/bin/bash";
```

Comment:
Nit: Recommend moving this after `info(command);` since it defines a variable used in the `try` block.